app/src/main/java/com/android/sample/ui/createRecipe/RecipeListInstructionsScreen.kt

```diff
@@ -102,12 +107,30 @@ fun RecipeListInstructionsContent(
         modifier = Modifier.height(BIG_PADDING.dp).testTag(RECIPE_LIST_INSTRUCTIONS_SCREEN_SPACER1))
 
     // This text represents the name of the recipe
-    Text(
-        modifier = Modifier.testTag(RECIPE_NAME_TEXT),
-        text = createRecipeViewModel.getRecipeName(),
-        style = MaterialTheme.typography.titleMedium,
-        color = MaterialTheme.colorScheme.onPrimary,
-    )
+    Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.SpaceBetween) {
+      Text(
+          modifier = Modifier.testTag(RECIPE_NAME_TEXT),
+          text =
+              if (createRecipeViewModel.getRecipeName().length > RECIPE_NAME_MAX_CHAR)
+                  createRecipeViewModel.getRecipeName().take(RECIPE_NAME_MAX_CHAR) + "..."
+              else createRecipeViewModel.getRecipeName(),
+          style = MaterialTheme.typography.titleMedium,
+          color = MaterialTheme.colorScheme.onPrimary,
+      )
+
+      Icon(
+          imageVector = Icons.Default.Add,
+          contentDescription = "Add",
```

Comment:
The `Icon` implementation is great, but moving the `contentDescription` to the `strings.xml` file would make it more accessible and manageable. This is especially helpful for users with assistive technologies, like screen readers, and improves inclusivity for diverse user needs.

Reply:
If you can apply it to the others contentDescription also it will be good

Reply:
Done !